profiles/common/modules/features/sitemap/README.md

```diff
@@ -0,0 +1,30 @@
+Sitemap Feature
+======================
+
+The module aims to provide a Sitemap (XML).
+
+# Installation
+
+The feature can be activated using feature set menu #
+(/admin/structure/feature-set).
+
+# Usage
+
+When feature is enabled, the /sitemap.xml is available.
+
+The feature provides:
+- list of XML sitemap,
+- Search engines,
+- Administration page (Setting),
+- Manually Rebuild links.
+
+## Regenerate the XML sitemap
+The feature regenerates the XML sitemap files using the cron or it's possible to
+regenerate manually.
+
+## Multilingual
+The feature includes the tags (rel = "alternate") for assuming the various
```

Comment:
"for assuming the various languages" is not clear to me what you mean with this, can this be clarified?